### ISSUE TITLE
GraphQL issue with "Entries" field (default value not working)

### ISSUE THREAD

**Issue by benediktblank** [verbb/formie#883]
**Description**
This issue is about the "Entries" element field.
You can assign a "default value" there. But when I try to query that value, it doesn't work because "defaultValue" is a string field.
But in that case a structure/array with the entry information would be returned, which doesn't work in the "defaultValue" field.

**Steps to reproduce**
1. Create a new form with the entries field and configure a "default value" for that field:
![image](https://user-images.githubusercontent.com/69143768/168090691-245c67ea-4b81-450e-bc17-3eef45f9c800.png)

2. Query the entries field and in particular the "defaultValue" field:
```
query MyQuery {
  formieForm {
    ... on Form {
      fields {
        ... on Field_Entries {
          id
          name
          defaultValue
        }
      }
    }
  }
}
```

The response is:
```
{
  "errors": [
    {
      "message": "String cannot represent value: [{\"id\":\"2754\",\"siteId\":\"1\"}]",
      "extensions": {
        "category": "graphql"
      }
    }
  ],
  "data": {
    "formieForm": {
      "fields": [
        {
          "id": "491",
          "name": "FormieEntries",
          "defaultValue": null
        }
      ]
    }
  }
}
```

### Form settings
Please pick from the options below.

- Multi-page form: No
- Submission Method: Ajax
- Client-side Validation: No
- Custom Form Templates: No

**Additional info**
- Plugin version: 1.5.16
- Craft version: 3.7.41
- Multi-site: No

**Comment by engram-design** [comment]
We've cleaned up  a bunch of things for GraphQL in Formie 2 (for Craft 4), including this handling. To get around this type collision, you'll need to use some other properties.

I've just added `defaultCategory`,  `defaultEntry`, `defaultProduct`, `defaultTag`, `defaultUser`, and `defaultVariant` to element field GraphQL queries which you should use instead.

Fixed for the next release. To get this fix early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-3 as 1.5.16",
  "...": "..."
}
```

Then run `composer update`.

**Comment by engram-design** [comment]
Fixed in [1.5.17](https://github.com/verbb/formie/releases/tag/1.5.17)
